@jbrowse/react-linear-genome-view not working because of @jbrowse/plugin-circular-view

<!-- Please fill out as much of the template as you can. -->

When creating a new react app and installing the @jbrowse/react-linear-genome-view it breaks with this error:

***Failed to compile.***

***./node_modules/@jbrowse/plugin-variants/dist/plugin-variants.esm.js
Module not found: Can't resolve '@jbrowse/plugin-circular-view'***

<!-- Put a clear and concise description of the bug here. -->

The steps to reproduce this bug are 
1. Create a react app using: npx create-react-app <project-name>
2. Install @jbrowse/react-linear-genome-view (1.0.1): npm i @jbrowse/react-linear-genome-view
2. In the auto-generated App.js folder change the contents by (following the tutorial): 

import React from 'react'
import 'fontsource-roboto'
import {
  createViewState,
  createJBrowseTheme,
  JBrowseLinearGenomeView,
  ThemeProvider,
} from '@jbrowse/react-linear-genome-view'
 
const theme = createJBrowseTheme()
 
function View() {
  defaultSession = {
    name: 'aNameHere', // what the name is doesn't really matter
    view: {/* view state here */},
  }
  const state = createViewState({
    assembly,
    tracks,
    defaultSession,
    location: 'ctgA:1105..1221',
    onChange: patch => {
      console.log('patch', patch)
    },
  })
  return (
   \<ThemeProvider theme={theme}>
      \<JBrowseLinearGenomeView viewState={state} />
    \</ThemeProvider>
  )
}

export default View

3. In the auto-generated replace: 
import App from './App'; with import View from './App';
and 
\<App /> with \<View />

--- a/products/jbrowse-react-linear-genome-view/README.md
+++ b/products/jbrowse-react-linear-genome-view/README.md
@@ -37,7 +37,7 @@ function View() {
 }
 ```
 
-For a full working example, see [this example](./docs/example).
+For a full working example, see [this example](docs/example.md).
 
 ## Install
 

--- a/products/jbrowse-react-linear-genome-view/package.json
+++ b/products/jbrowse-react-linear-genome-view/package.json
@@ -12,7 +12,7 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/GMOD/jbrowse-components.git",
-    "directory": "packages/jbrowse-react-linear-genome-view"
+    "directory": "products/jbrowse-react-linear-genome-view"
   },
   "author": "JBrowse Team",
   "main": "dist/index.js",
@@ -36,6 +36,7 @@
     "@jbrowse/core": "^1.0.2",
     "@jbrowse/plugin-alignments": "^1.0.2",
     "@jbrowse/plugin-bed": "^1.0.2",
+    "@jbrowse/plugin-circular-view": "^1.0.2",
     "@jbrowse/plugin-config": "^1.0.2",
     "@jbrowse/plugin-data-management": "^1.0.2",
     "@jbrowse/plugin-gff3": "^1.0.2",

